src/main/java/edu/umich/tl/CourseDelete.java

```diff
@@ -66,16 +80,45 @@ private static void readPropertiesFromFile(String[] args) {
 		String propFileLocation;
 		if(args.length == 0) {
 			M_log.error("Command line arguments are not provided");
-			System.exit(0);
+			System.exit(1);
 		}
 		propFileLocation = args[0];
 		properties = Utils.getPropertiesFromURL(propFileLocation);
 		canvasToken = properties.getProperty(CANVAS_TOKEN);
 		canvasURL = properties.getProperty(CANVAS_URL);
 		esbURL=properties.getProperty(ESB_URL);
-		sisTermIdFromPropsFile = properties.getProperty(SIS_TERM_ID);
+		getTermsInfoFromProperties(properties);
 		checkForApiType(properties);
 	}
+	/*
+	 * The Terms that needs for course deletion are listed in the properties file. Each term has 2 piece of information 
+	 * term1=2030;04/06/15, the sisTermCode and the endDate of that term. We will check if the current system date is
+	 * past the endDate then that term becomes eligible for deleting the unused/unpublished courses that are created 
```

Comment:
It's how ever BSA defines it what date he wants to put in. Their is lot of confusion about end date from different systems. Mpathways has their own term end date, canvas has some floating end dates to keep the courses alive in the canvas system. To me it is the date after which the courses will be consider for deletion.